packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.html

```diff
@@ -4,136 +4,189 @@
     </c-quantic-component-error>
   </template>
   <template lwc:else>
-    <template if:true={showPlaceholder}>
-      <c-quantic-placeholder variant="card" number-of-rows={numberOfValues}></c-quantic-placeholder>
-    </template>
-    <template if:true={hasParentsOrValues}>
-      <div class="slds-size_1-of-1 slds-var-m-bottom_large">
-        <c-quantic-card-container title={label} onquantic__headerclick={toggleFacetVisibility} onquantic__headerkeydown={toggleFacetVisibility}>
-          <lightning-button-icon
-            class={actionButtonCssClasses}
-            slot="actions"
-            onmousedown={preventDefault}
-            icon-name={actionButtonIcon}
-            alternative-text={actionButtonLabel}
-            variant="bare"
-            tabindex="-1"
-            aria-hidden="true"
+    <template lwc:if={enabled}>
```

Comment:
What is enabled here? the facet? I think we should name this something a bit more descriptive.